kiliautoml/utils/ultralytics/train.py

```diff
@@ -167,6 +178,7 @@ def ultralytics_train_yolov5(
         project_id=project_id,
         max_assets=max_assets,
         assets_status_in=asset_status_in,
+        randomize_assets=randomize_assets,
```

Comment:
the yaml preparation does not require so many arguments now, so that's a good time to do some clean-up: only the 3 first ones are needed (you can remove the following ones, and remove the unnecessary arguments in the call hierarchy)